security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,21 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-0"]
+== OpenShift File Integrity Operator 1.3.0
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:3600[RHBA-2023:3600 OpenShift File Integrity Operator Bug Fix Update]
+
+* FIO now includes kubelet certificates as default files, excluding them from throwing warnings when they're managed by OpenShift.
+
+* The operator now provides the ability to accommodate the email address for Red Hat Technical Support.
+
+* Previously, FIO would not clean up node status CRDs when nodes are removed from the cluster. FIO would also erroneously indicate that new nodes failed integrity checks. FIO has been updated to correctly clean up node status CRDs on node removal or when adding new nodes to the cluster. This provides correct node status notifications.
```

Comment:
Don't forget to add the URL to the jira at the end. For example (from the Compliance Operator relnotes):
```
* Previously, .... (etc) .... upgrades from any previous version will not result in a missing SA. (link:https://issues.redhat.com/browse/OCPBUGS-3452[*OCPBUGS-3452*])
```

Reply:
Wow yes, thank you, added.